src/main/java/storagesystem/viewcontroller/inventory/InventoryListItemController.java

```diff
@@ -0,0 +1,100 @@
+package storagesystem.viewcontroller.inventory;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import javafx.scene.layout.AnchorPane;
+import storagesystem.model.IReservable;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Pär Aronsson
+ */
+
+public class InventoryListItemController extends AnchorPane {
+
+    public Label getName() {
+        return name;
+    }
+
+    public Label getAmount() {
+        return amount;
+    }
+
+    public Label getLocation() {
+        return itemLocation;
+    }
+
+    public Label getCondition() {
+        return condition;
+    }
+
+    @FXML
+    private AnchorPane rootPane;
+
+    @FXML
+    private Label name;
+
+    @FXML
+    private Label amount;
+
+    @FXML
+    private Label itemLocation;
+
+    @FXML
+    private Label condition;
+
+    IReservable thisItem;
```

Comment:
Methods should be under attributes. Getters/setters at bottom of class aswell since they are not as interesting as other methods.